jscomp/main/jsoo_main.mli

```diff
@@ -22,4 +22,5 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA. *)
 
-val make_compiler : string -> (Lexing.lexbuf -> Parsetree.structure) -> unit
+val make_compiler :
+  string -> (Lexing.lexbuf -> Melange_compiler_libs.Parsetree.structure) -> unit
```

Comment:
we could `-open Melange_compiler_libs` in flags for `jsoo_main` like we do everywhere else

Reply:
https://github.com/melange-re/melange/pull/538/commits/1620cbc9024e36c83775c7f4ba2dc7461d2fddf4